apps/web/app/meetings/_components/Reservation/ReservationForm.tsx

```
@@ -35,21 +65,94 @@ export default function ReservationForm(props: ReservationFormProps): JSX.Elemen
+  }, [startTimeValue, customStartTimeValue, setValue, selectedTime]);
+
+  useEffect(() => {
+    const compareTimes = (start: string, end: string): boolean => {
+      const [startHour = 0, startMinute = 0] = start.split(":").map((value) => {
+        const num = parseInt(value, 10);
+        return isNaN(num) ? 0 : num;
+      });
+
+      const [endHour = 0, endMinute = 0] = end.split(":").map((value) => {
+        const num = parseInt(value, 10);
+        return isNaN(num) ? 0 : num;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)